src/constants.js

```diff
@@ -0,0 +1,2 @@
+const URL = ' https://wa.me/526145153963';
+export default URL;
```

Comment:
this file is suposed to export multiple constants at one, don't use export default
use this instead
```
export const URL = ...
```